### PR TITLE
Update remote node and bootsrap node guide

### PR DESCRIPTION
**Pull request by SamsungGalaxyPlayer**

### CODE REVIEW

**Comment by SamsungGalaxyPlayer** [comment]
Closes #103.

File: en/ch08.md [hunk 1]
```diff
@@ -1,7 +1,5 @@
 # About remote nodes
-Remote nodes can be very useful if you are not able/don't want to download the whole blockchain, but be advised that malicious remote nodes could compromise the level of privacy of your transaction, tracking your IP or, in extreme cases, even showing the amount transacted. That said, a list of remote nodes can be found at [moneroworld.com](https://moneroworld.com). Keep in mind anybody is able to add a node to that list, you shouldn't consider those nodes as trusted or safe, always run your own node for the best privacy.
+Remote nodes are useful if you are not able/don't want to download the whole blockchain, but be advised that malicious remote nodes could compromise some privacy. They can track your IP address, track your "restore height" and associated block request data, and send you inaccurate information to learn more about transactions you make. Please make sure to use a node you trust (hopefully your own) and remain aware of these limitations. Users who prioritize privacy should use a full node instead. A number of open nodes are listed at [moneroworld.com](https://moneroworld.com) and [node-o-matic](https://node.pwned.systems)
```
Comment (xmrdsc) [inline en/ch08.md]: `they can` -> `they could`

Glass half full!

Comment (SamsungGalaxyPlayer) [reply en/ch08.md]: Changed to "may" to be consistent with the other paragraph

Comment (xmrdsc) [reply en/ch08.md]: :+1:

**Pull request merged**